javascript/node/pnpm-lock.yaml

```diff
@@ -0,0 +1,226 @@
+lockfileVersion: '9.0'
```

Comment:
I realized I sent you in the wrong direction with `pnpm`, it was really NPM that should be used, sorry.

Need to be called within the `node` folder.